project1/css/index.css

```diff
@@ -62,3 +62,95 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.container {
+  width: 85%;
+	margin: auto;
+}
+  
+  header {
+	display: flex;
+	width: 100%;
+}
+  
+  header nav ul {
+	display: flex;
+	list-style: none;
+	justify-content: space-around;
+}
+  
+  header nav {
+	width: 50%;
+	flex-grow: 1
+}
+  
+  header nav ul li a {
+	text-decoration: none;
+	color: grey;
+}
+  
+  header .logo {
```

Comment:
Making the logo 50px x 50 px makes it look very pixelated and odd, I recommend leaving it at it's default res and finding a better solution for fitting it on the top row. If this were for a company for instance, they wouldn't like the logo getting squished into the top corner!